src/Microsoft.Health.Dicom.Core/Features/ExtendedQueryTag/IndexTag.cs

```diff
@@ -20,29 +20,29 @@ public class IndexTag
         /// <remarks>Used for constuctoring from core dicom tag.PatientName e.g. </remarks>
         /// <param name="tag">The core dicom Tag.</param>
         /// <param name="level">The tag level.</param>
-        public IndexTag(DicomTag tag, CustomTagLevel level)
```

Comment:
I'm thinking if we could rename it as _QueryTag_ or _QueryableTag_ for consistency, so that could be easier to understand under the context of ExtendedQueryTag